src/main/java/uk/gov/hmcts/darts/usermanagement/exception/UserManagementError.java

```diff
@@ -23,7 +23,11 @@ public enum UserManagementError implements DartsApiError {
     DUPLICATE_SECURITY_GROUP_NAME_NOT_PERMITTED(
         UserManagementErrorCode.DUPLICATE_SECURITY_GROUP_NAME_NOT_PERMITTED.getValue(),
         HttpStatus.CONFLICT,
-        UserManagementTitleErrors.DUPLICATE_SECURITY_GROUP_NAME_NOT_PERMITTED.toString()
+        UserManagementTitleErrors.DUPLICATE_SECURITY_GROUP_NAME_NOT_PERMITTED.toString()),
```

Comment:
Should these error descriptions be duplicate security or invalid email?

Reply:
not changed these, only added the new one - INVALID_EMAIL_FORMAT